src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -375,8 +375,14 @@ struct SpecifiedStartTime : Tags::StartTime, db::SimpleTag {
   using option_tags = tmpl::list<OptionTags::StartTime>;
 
   static constexpr bool pass_metavariables = false;
-  static double create_from_options(const double start_time) noexcept {
-    return start_time;
+  static double create_from_options(
+      const std::optional<double> start_time) noexcept {
+    if(not static_cast<bool>(start_time)) {
```

Comment:
I'm happy for this to be changed in a followup PR: we've been preferring `start_time.has_value()` over the bool cast. I think `boost::optional` didn't have a `has_value()`, which is why we used to have the cast.